src/main/java/org/amoseman/budgetingbackend/resource/AccountResource.java

```diff
@@ -46,8 +47,8 @@ public Response removeAccount(@Auth User user, @PathParam("username") String use
     @Consumes(MediaType.APPLICATION_JSON)
     @Path("/password")
     public Response changePassword(@Auth User user, String password) throws AccountDoesNotExistException {
-        accountService.changePassword(user.getName(), password);
-        return Response.ok().build();
+        ResultType result = accountService.changePassword(user.getName(), password);
+        return Response.ok(result).build();
```

Comment:
Same problem here, there should be a shared method that would build the right Response for the ResultType.